pages/core-api/api-safe-transaction-service.mdx

```diff
@@ -166,45 +166,8 @@ Supported and configured networks on `safe-eth-py`:
 - [**Etherscan** configured networks](https://github.com/safe-global/safe-eth-py/blob/master/gnosis/eth/clients/etherscan_client.py#L24)
 - [**Blockscout** configured networks](https://github.com/safe-global/safe-eth-py/blob/master/gnosis/eth/clients/blockscout_client.py#L21)
 
-### Endpoints
-
-- `POST /v1/data-decoder/` decodes a transaction `data` passed on the body for a `to` contract address.
-
-**Example transaction decoder**
-
-```bash
-curl -X 'POST' \
-  'https://safe-transaction-sepolia.safe.global/api/v1/data-decoder/' \
-  -H 'accept: application/json' \
-  -H 'Content-Type: application/json' \
-  -H 'X-CSRFToken: Gx1aRa8kIJGIAfReLAWwr9Q6dHv22dFt7VprdipLryHcxpfhk9aV0UDAhNz8gGYz' \
-  -d '{
-  "data": "0x095ea7b3000000000000000000000000e6fc577e87f7c977c4393300417dcc592d90acf8ffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff",
-  "to": "0x4127839cdf4F73d9fC9a2C2861d8d1799e9DF40C"
-}'
-```
-
-Output:
-
-```
-{
-  "method": "approve",
-  "parameters": [
-    {
-      "name": "spender",
-      "type": "address",
-      "value": "0xe6fC577E87F7c977c4393300417dCC592D90acF8"
-    },
-    {
-      "name": "value",
-      "type": "uint256",
-      "value": "115792089237316195423570985008687907853269984665640564039457584007913129639935"
-    }
-  ]
-}
-```
 
-This decoded data is also included as `dataDecoded` in the `GET /multisig-transactions`, `GET /module-transactions` and `GET /all-transactions` endpoints. 
+The following [Guides](../core-api/transaction-service-guides/data-decoder.mdx) show how to decode contract interactions data of a transaction from the Safe Transaction Service.
```

Comment:
```suggestion
Follow our [guides](https://add_data_decoder_guide--docs.review.5afe.dev/core-api/transaction-service-guides/data-decoder) to learn how to decode contract interaction data using the Safe Transaction Service for a transaction.
```

Reply:
Updated here https://github.com/safe-global/safe-docs/pull/489/commits/c17918ee6926fd038220681598adada7cefdff80 because the link is incorrect. Maybe we can do the same update on lines 154 and 126

Reply:
Duh, my bad @falvaradorodriguez! Makes sense on the same update to the lines you mentioned.

Reply:
Done here https://github.com/safe-global/safe-docs/pull/489/commits/5bf8839ea59c4d7e796a356e68b4a2445e3c0ae6. Thanks @tanay1337